index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simple-singolo</title>
+    <link rel="stylesheet" media="screen" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="header">
+            <h1 class="logo">singolo<span class="star">*</span></h1>
+            <nav class="navigation">
+                <div class="header-link-menu">
+                    <a class="navigation-links" href="#"><span>HOME</span></a>
+                    <a class="navigation-links" href="#"><span>SERVICES</span></a>
+                    <a class="navigation-links" href="#"><span>PORTFOLIO</span></a>
+                    <a class="navigation-links" href="#"><span>ABOUT</span></a>
+                    <a class="navigation-links" href="#"><span>CONTACT</span></a>
+                </div>
+            </nav>
+        </div>
+    </header>
+
+       <main>
+        <section class="main-screen">
+
+            <div class="left-arrow-block">
+                <a href="#"><img class="arrow left-arrow" src="img/arrow-left.svg" alt="left arrow"></a>
+            </div>
+            <div class="completely-vertical">
+                <img class="shadow-vertical" src="/img/phone-vertical-shadow.svg" alt="vertical shadow">
```

Comment:
I missed a lit of images. Please check that all images has relative path './img/phone....' or 'img/phone...'